src/main/java/com/amihaiemil/camel/YamlLine.java

```diff
@@ -80,5 +75,18 @@ public int indentation() {
             return 0;    
         }
         
+        @Override
+        public int compareTo(final YamlLine other) {    
+            int result = -1;
```

Comment:
Here just ``return -1``, since a NullYamlLine, which is empty, will always be alphabetically first, before any other line.